hw/ip/otbn/doc/_index.md

```diff
@@ -568,10 +568,16 @@ A fatal alert can only be cleared by resetting OTBN through the `rst_ni` line.
 ### Idle
 
 OTBN exposes a single-bit `idle_o` signal, intended to be used by the clock manager to clock-gate the block when it is not in use.
-This signal is high when OTBN is not running.
+This signal is in the same clock domain as `clk_i`.
+It is high when OTBN is not running.
 The cycle after a write to {{< regref "CMD.start" >}}, the signal goes low.
 This remains low until the end of the operation (either from an {{< otbnInsnRef "ECALL" >}}) or an error, at which point it goes high again.
 
+OTBN also exposes another version of the idle signal as `idle_otp_o`.
+This works analogously, but is in the same clock domain as `clk_otp_i`.
+
+TODO: Specify interactions between `idle_o`, `idle_otp_o` and the clock manager fully.
```

Comment:
Why mention the otp block here?

Reply:
Because this idle signal is used for the (transactional) clock that we use for our connection to the OTP block.